plugin/plugin.go

```diff
@@ -212,7 +212,7 @@ func (p *Plugin) renderValidatorObjectMethods() {
 
 func (p *Plugin) renderValidatorObjectMethod(o *descriptor.DescriptorProto, t string) {
 	p.P(`// validate_Object_`, t, ` function validates a JSON for a given object.`)
-	p.P(`func validate_Object_`, t, `(r json.RawMessage, path string, allowUnknown bool) (err error) {`)
+	p.P(`func validate_Object_`, t, `(ctx context.Context,r json.RawMessage, path string, allowUnknown bool) (err error) {`)
 	p.P(`obj := &`, t, `{}`)
 	p.P(`if hook, ok := interface{}(obj).(interface { AtlasJSONValidate(json.RawMessage, string, bool) (json.RawMessage, error) }); ok {`)
```

Comment:
context missing

Reply:
In this pull request i don't break custom validate method contract, in next pull request i move all data to context and add context to custom validate method

Reply:
ok